### PR TITLE
Fixes issue when bozz appears on first load

### PR DESCRIPTION
**Pull request by naomilwx**
When bozz is the first slide, the `playToggle` call gets aborted, hence the `is_paused` variable is not set to false. As a result, the slides do not auto-advance.

To fix this, `is_paused` is set to false in `goToRandomGuaranteed`.

The indentation of the original supersized library is terrible. Fixed that and also removed some of the oneline if statements because they are hard to read.

### CODE REVIEW

File: js/supersized-karaoke.js [hunk 1]
```diff
@@ -1,955 +1,983 @@
...
+      return false;
+
+    };
+
+      /* Go to random guaranteed slide
+    ----------------------------*/
+    base.goToRandomGuaranteed = function() {
+      // base.goTo() starts counting from 1
+      slideIndex = Math.ceil(Math.random() * guaranteedSlides.length + vars.original_slide_length);
+      console.log('bozz slide number: ' + slideIndex);
```
Comment (fonglh) [inline js/supersized-karaoke.js]: Change this log message, then we can contribute this back upstream.

Comment (naomilwx) [reply js/supersized-karaoke.js]: Hmm I don't think I added this...I think this log line is already upstream :joy:

Comment (fonglh) [reply js/supersized-karaoke.js]: yeah it was from my accidental PR. but let's get rid of this upstream too.

Comment (naomilwx) [reply js/supersized-karaoke.js]: ok removed all bozz references

**Pull request merged**